src/services/routing.ts

```diff
@@ -24,23 +24,16 @@ export enum PageSlugs {
   fields = 'fields',
 }
 
-export function encodeParameter(parameter: string): string {
-  return encodeURIComponent(parameter.replace(/\//g, '---'));
-}
-
-export function decodeParameter(parameter: string): string {
-  return decodeURIComponent(parameter).replace(/---/g, '/');
-}
-
 export const ROUTES = {
   explore: () => prefixRoute(PageSlugs.explore),
-  logs: (service: string) => prefixRoute(`${PageSlugs.explore}/service/${encodeParameter(service)}/${PageSlugs.logs}`),
+  logs: (service: string) =>
+    prefixRoute(`${PageSlugs.explore}/service/${service.replace(/\//g, '-')}/${PageSlugs.logs}`),
   fields: (service: string) =>
-    prefixRoute(`${PageSlugs.explore}/service/${encodeParameter(service)}/${PageSlugs.fields}`),
+    prefixRoute(`${PageSlugs.explore}/service/${service.replace(/\//g, '-')}/${PageSlugs.fields}`),
   patterns: (service: string) =>
-    prefixRoute(`${PageSlugs.explore}/service/${encodeParameter(service)}/${PageSlugs.patterns}`),
+    prefixRoute(`${PageSlugs.explore}/service/${service.replace(/\//g, '-')}/${PageSlugs.patterns}`),
   labels: (service: string) =>
-    prefixRoute(`${PageSlugs.explore}/service/${encodeParameter(service)}/${PageSlugs.labels}`),
+    prefixRoute(`${PageSlugs.explore}/service/${service.replace(/\//g, '-')}/${PageSlugs.labels}`),
```

Comment:
I just realized now, but we could have implemented the replace within `encodeParameter()` so you don't need to `service.replace()` every route.